src/views/DashboardView/data.js

```diff
@@ -1,6 +1,39 @@
-const initData = `
+import { v4 as uuid } from 'uuid';
+
+export const todoData = `
 ### To Do:
  - Hurrah! Nothing left to do!
 `;
 
-export default initData;
+export const trelloData = {
```

Comment:
So name it `trelloPlaceHolderData` ? 

Coz there will be other placeholder data for other modals. Should we rename `data.js` to `placeHolders.js`?

Reply:
> So name it trelloPlaceHolderData ?

I'm kinda against using the word "Trello" in our code since it's proprietary. We could replace Trello with `kanban` instead, for example name this to `kanbanPlaceHolderData`.

> Should we rename data.js to placeHolders.js?

Makes sense.

Reply:
Well, we could go for `kanban` if were to keep the `Trello import` out of the scoped plan.

I think using Trello should be fine here, as we are directly associating our kanban with Trello and allowing imports

Reply:
I see `placeHoldergen` instead of `placeHolders` @sauravhiremath . Any specific reason?

Reply:
Just avoiding the use of plurals to keep naming consistent

Reply:
Then I think `placeHolder.js` would be better. placeHolderGen sounds like something German haha

Reply:
Fixed in a6ce06c